backend/src/service/user_data.rs

```diff
@@ -0,0 +1,24 @@
+//! Service layer for user data.
+
+use actix_web::web::Data;
+use uuid::Uuid;
+
+use crate::{
+    config::data::AppDataInner,
+    error::ServiceError,
+    model::{dto::UserDataDto, entity::UserData},
+};
+
+/// Create an user data entry for a new user.
```

Comment:
```suggestion
/// Create a user data entry for a new user.
```
User is pronounced with an initial consonant ('juser').
Therefore 'a' has to be used instead of 'an'.